spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -311,6 +325,210 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
+            min_dorsal_bridge_width_slice = df_temp.loc[df_temp['dorsal_bridge_width'].idxmin(), 'axial_slice']
+            # Get the axial slice with the minimum ventral tissue bridge for the selected sagittal slice
+            min_ventral_bridge_width_slice = df_temp.loc[df_temp['ventral_bridge_width'].idxmin(), 'axial_slice']
+
+            # Add a new column with value True to tissue_bridges_df; this information is needed for plotting
+            tissue_bridges_df.loc[(tissue_bridges_df['sagittal_slice'] == sagittal_slice) &
+                                  (tissue_bridges_df['axial_slice'] == min_dorsal_bridge_width_slice), 'min_dorsal_bridge_axial_slice'] = True
+            tissue_bridges_df.loc[(tissue_bridges_df['sagittal_slice'] == sagittal_slice) &
+                                  (tissue_bridges_df['axial_slice'] == min_ventral_bridge_width_slice), 'min_ventral_bridge_axial_slice'] = True
+            # Replace NaN with False
+            tissue_bridges_df.fillna(False, inplace=True)
+
+            # Save info for plotting
+            plot_dict[sagittal_slice, 'dorsal'] = df_temp[df_temp['axial_slice'] == min_dorsal_bridge_width_slice]
+            plot_dict[sagittal_slice, 'ventral'] = df_temp[df_temp['axial_slice'] == min_ventral_bridge_width_slice]
+
+            # Get the width of the tissue bridges in mm
+            # NOTE: the orientation is RPI (because we reoriented the image to RPI using orient2rpi()); therefore
+            # p_lst[0] is the pixel size in the R-L direction, p_lst[1] is the pixel size in the A-P direction, and
+            # p_lst[2] is the pixel size in the S-I direction.
+            # Since we are computing dorsal and ventral tissue bridges, we use p_lst[1] (A-P direction)
+            # NOTE: we use np.cos(self.angles[SLICE]) to correct for the angle of the spinal cord with respect to the
+            # slice
+            dorsal_bridge_width_mm = (float(df_temp[df_temp['axial_slice'] ==
+                                                    min_dorsal_bridge_width_slice]['dorsal_bridge_width'] * p_lst[1]) *
+                                      np.cos(self.angles[min_dorsal_bridge_width_slice]))
+            ventral_bridge_width_mm = (float(df_temp[df_temp['axial_slice'] ==
+                                                     min_ventral_bridge_width_slice]['ventral_bridge_width'] * p_lst[1]) *
+                                       np.cos(self.angles[min_ventral_bridge_width_slice]))
+            total_bridge_width_mm = dorsal_bridge_width_mm + ventral_bridge_width_mm
+
+            # Save the minimum tissue bridges
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_dorsal_bridge_width [mm]'] = dorsal_bridge_width_mm
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_ventral_bridge_width [mm]'] = ventral_bridge_width_mm
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_total_bridge_width [mm]'] = total_bridge_width_mm
```

Comment:
Since we create a new column for every sagittal slice, what happens if we have multiple lesions spanning different sagittal slice ranges?

I guess in that case, there will be quite a lot of columns (3 per sagittal slice), but some of the cells will be blank for different lesions (since not all lesions span all slices).

I'm wondering if there is a cleaner way to organize the sheet. I am reminded of the `-f` option of `sct_analyze_lesion`, which produces 1 sheet page per lesion. In that case, we could have "sagittal slice" be one axis, and "dorsal/ventral/total" be the other axis.

(Though, this would mean that the bridge width data would be separate from the other measurements.)

Reply:
> Since we create a new column for every sagittal slice, what happens if we have multiple lesions spanning different sagittal slice ranges?

Very good point! Tbh, I kind of simplified the problem now only for a single lesion because tissue bridges are currently computed exclusively in spinal cord injury (SCI), which is characterized by a single lesion. But the truth is that `sct_analyze_lesion` can also be used for MS patients with multiple lesions... The question is, how clinically relevant are tissue bridges in MS?

Reply:
> tissue bridges are currently computed exclusively in spinal cord injury (SCI), which is characterized by a single lesion.

This is very important context! Thank you for clarifying.

Maybe since we only want to compute this measure for one specific case, it could be opt-in? (As to not overcomplicate the output if the user has no use for the measure.) And, if we had a CLI argument, then the help description would be a good place where we can document the option (by explaining the relevance for SCI + linking the relevant citation).

Reply:
Okay! Even tSCI patients might have multiple lesions (see the last screenshot in [this comment](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489#issuecomment-2168479780)) --> update the script to handle multiple lesions properly.

Reply:
Okay, it seems that the code is already capable of handling multiple lesions! In such a case, statistics (including tissue bridges) for each lesion are stored as individual rows in the output XLS file, for example:

<img width="2146" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/167e3ef0-110d-4c12-9f50-2ff9b5b828e0">

I just added this [small commit](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489/commits/f030fbad8b13327eef2bcef1ab44d1e273acf23f) to replace `nan` with `n/a` (`nan` results in empty cells in Excel).

Just the [plotting of tissue bridges](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/f030fbad8b13327eef2bcef1ab44d1e273acf23f/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L479-L549) is not ready for multiple lesions --> I will update it!